src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int GROW_NUMBER = 2;
+    private Node<K, V>[] table;
+    private int threshold;
```

Comment:
`threshold` in redundant variable, you can simple use `table.length * LOAD_FACTOR` where it is needed